vsc_pdf_template_editor/lib/src/stores/tree_store.dart

```diff
@@ -84,10 +97,11 @@ abstract class TreeStoreModel with Store {
   onInputChanged() {
     try {
       _template = jsonDecode(_templateController.rawText);
-      _buildPdf();
       buildErrorText = '';
```

Comment:
I fixed the position where `buildErrorText` was being cleared - it was not reporting errors sometimes because it was cleared after the error occurred.